physionet-django/console/templates/console/submitted_projects.html

```diff
@@ -118,6 +119,12 @@ <h6 id='project_name'></h6>
                       Not a resubmission
                     {% endif %}
                   </td>
+                  {% if project.editor == user %}
+                    <td><a class="btn btn-primary btn-sm" href="{% url 'edit_submission' project.slug %}" role="button">Edit Project</a></td>
+                  {% else %}
+                    <td><p>Ongoing</p></td>
```

Comment:
Any chance this could say "Assigned to <editor name>" instead?

Reply:
Actually scrap that...I think it's okay because we have the editor in separate column.